nextcord/interactions.py

```diff
@@ -359,6 +364,43 @@ async def delete_original_message(self) -> None:
             session=self._session,
         )
 
+    async def send(self, *args, **kwargs) -> Optional[Message]:
+        """|coro|
+
+        This is a shorthand function for helping in sending messages in
+        response to an interaction. If the response
+        :meth:`InteractionResponse.is_done()` then the message is sent
+        via the :attr:`Interaction.channel` instead.
+
+        .. warning::
+
+            Ephemeral messages should not be sent with this as if
+            the :attr:`Interaction.channel` is fallen back to,
+            ephemeral messages cannot be sent with this.
+
+        Returns
+        -------
+        Optional[:class:`Message`]
+            Message if the interaction has been responded to and the
+            interaction's channel was sent to. Else ``None``
+
+        Raises
+        ------
+        InvalidData
+            Somehow :attr:`Interaction.channel` was ``None``,
+            this may occur in threads.
+        """
+
+        if not self.response.is_done():
+            await self.response.send_message(*args, **kwargs)
+        else:
+            if self.channel is not None:
+                await self.channel.send(*args, **kwargs)
+            else:
+                raise InvalidData(
+                    "Interaction.channel is None, this may occur in threads"
+                )
+
 
```

Comment:
This is a personal thing, so feel free to ignore
```python
if not self.response.is_done():
    return await self.response.send_message(*args, **kwargs)

if self.channel is not None:
    return await self.channel.send(*args, **kwargs)

raise InvalidData(
    "Interaction.channel is None, this may occur in threads"
)
```